recipes/pato/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pato" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jmeurisse/pato_releases_conda/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f4bf78d600f48f56382c93c758deb882b3600704080ae0a0942596405df1bef8
+  
+build:
+  number: 0
+  skip: True  # [win or linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp
+    - make
+    - wget
+    - cmake 
+    - boost 
+    - metis
+    - cgal
+    - scotch
+
+  host:
+    - qt
+    - openmpi
```

Comment:
Please see [the doc section on MPI support]( https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi )